net/nmap/Makefile

```diff
@@ -26,6 +26,7 @@ PKG_CPE_ID:=cpe:/a:nmap:nmap
 
 PKG_BUILD_PARALLEL:=1
 PKG_INSTALL:=1
+PYTHON3_PKG_BUILD:=0
```

Comment:
AFAICT, this is not quite needed, since `Build/Compile` is needed;
still fine to leave it;